optimizely/event/event_processor.py

```diff
@@ -60,7 +61,8 @@ def __init__(self,
                 event_queue=None,
                 batch_size=None,
                 flush_interval=None,
-                timeout_interval=None):
+                timeout_interval=None,
+                notification_center=None):
```

Comment:
Can you add some good explanations for all the fields here. Something like: https://github.com/optimizely/python-sdk/blob/master/optimizely/optimizely.py#L42-L59